### PR TITLE
Addresses BACKLOG-37841, BACKLOG-38006: Change schedule ownership

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon @dcleao please review

The colors added to `base.css` came from [DS dawn theme](https://zeroheight.com/6c705d900/p/56ed6f-dawn-theme/b/8801f3)
And the new icons from [DS icons lib](https://zeroheight.com/6c705d900/p/31e481-icons/b/75d9df)

### CODE REVIEW

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java [hunk 1]
```diff
@@ -330,17 +338,36 @@ public void addItem( String label ) {
...
   }
 
+  /**
+   * Returns a list of current ListItems that match {@code search}.
+   *
+   * @param search - value to filter ListItems
+   *
+   * @return list of filtered ListItems
+   */
+  public List<ListItem> getItems( String search ) {
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java:361]: Imo, `findItems` would be a more appropriate name.

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/text/SearchTextBox.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  private final TextBox input = new TextBox();
+  private final Image icon = ImageUtil.getThemeableImage( "icon-zoomable" );
+
+  public SearchTextBox() {
+    super();
+
+    createUI();
+  }
+
+  public TextBox getInput() {
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/gwt/widgets/client/text/SearchTextBox.java:49]: Needs to be public?

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/base.css [hunk 3]
```diff
@@ -50,6 +50,35 @@
...
+  --color-secondary: #414141;
+  --color-secondary-80: #6C6B6B;
+  --color-secondary-60: #999999;
+  --color-brand: #CC0000;
+
+  /* Semantic colors - Status & system feedback colors */
+  --color-neutral: #4D8AC0;
+  --color-positive: #478B1A;
+  --color-warning: #F9C846;
+  --color-negative: #D43136;
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/base.css:81]: 😍🚀

**Review by dcleao — CHANGES_REQUESTED** [review]
This is great, you never disappoint!

Well, just some nit picking.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/base.css (hunk 3 above)
Comment (davidmsantos90) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/base.css:81]: Just updated the PR description, to include links to the DS page with this colors specs 👍🏼

**Review by dcleao — APPROVED** [review]

**Review by afrjorge — APPROVED** [review]

**Review by mbrasil — APPROVED** [review]

**Comment by davidmsantos90** [comment]
Rebased and changed `setSelectedItemPlaceholderText` back to `public`. The schedule plugin needs to change the placeholder text in one its dialogs

**Review by Pancho7 — APPROVED** [review]

**Pull request merged**